controller/space.go

```diff
@@ -409,55 +409,75 @@ func ConvertSpaceFromModel(ctx context.Context, db application.DB, request *http
 			Version:     &sp.Version,
 		},
 		Links: &app.GenericLinksForSpace{
-			Self:    &relatedURL,
-			Related: &relatedURL,
-			Backlog: &app.BacklogGenericLink{
-				Self: &relatedBacklogList,
+			Self:    &selfURL,
+			Related: &selfURL, //TODO (xcoulon): remove this link
+			Backlog: &app.BacklogGenericLink{ //TODO (xcoulon): remove this link
+				Self: &relatedBacklog,
 				Meta: &app.BacklogLinkMeta{TotalCount: count},
 			},
-			Workitemtypes:      &relatedWorkItemTypeList,
-			Workitemlinktypes:  &relatedWorkItemLinkTypeList,
-			Filters:            &relatedFilterList,
-			Workitemtypegroups: &workitemTypeGroupsLink,
+			Workitemtypes:      &relatedWorkItemTypes,      //TODO (xcoulon): remove this link
+			Workitemlinktypes:  &relatedWorkItemLinkTypes,  //TODO (xcoulon): remove this link
+			Filters:            &relatedFilters,            //TODO (xcoulon): remove this link
+			Workitemtypegroups: &relatedWorkitemTypeGroups, //TODO (xcoulon): remove this link
```

Comment:
I suppose this TODO note is due once all clients of this API use the new and correct fields, right?